libsrc/geos-cbm/setnewmode.s

```diff
@@ -0,0 +1,18 @@
+;
+; Scott Hutter
+;
+; 18.12.18
+
+; void SetNewMode(void);
+
+            .export _SetNewMode
+
+            .include "jumptab.inc"
+            .include "geossym.inc"
+        
+_SetNewMode:
+			lda graphMode
```

Comment:
Please one use space chars, no tab chars in source code for the cc65 repo.

Reply:
got it